src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/actions.ts

```diff
@@ -24,6 +24,8 @@ import { CONST_MAX_NUM_ADDRESSES } from "../../../../../../../constants";
 import { SanitizedEmailAddressRow } from "../../../../../../functions/server/sanitize";
 import { deleteAccount } from "../../../../../../functions/server/deleteAccount";
 import { cookies } from "next/headers";
+import { applyCurrentCouponCode } from "../../../../../../functions/server/applyCoupon";
+import { checkCouponForSubscriber } from "../../../../../../../db/tables/subscriber_coupons";
```

Comment:
Is there a reason to import from the table instead of from /functions/server? There's a wrapper already for this function, you shouldn't have to recreate it?